Makefile

```diff
@@ -254,13 +248,19 @@ dockerfree-pause:
 dockerfree-certs:
 	./scripts/get-host-certs
 
-dockerfree-cni-plugins: get-cni-sources
+dockerfree-cni-plugins:
 	./scripts/build-cni-plugins
 
 # see dockerfree-pause above: assumes that the pre-compiled pause container tar exists
-dockerfree-agent-image: dockerfree-certs dockerfree-cni-plugins static-with-pause
+# builds agent image and saves on disk, assumes cni plugins have been pulled
+release-agent-internal: dockerfree-certs dockerfree-cni-plugins static
```

Comment:
When do we need this build target? Is there harm to always run `get-cni-sources`?

Reply:
Internal build process restricts connectivity to fetch submodules from github. This target facilitates building artifacts by assuming cni plugins have already been fetched and agent can be built in an isolated environment